governance/votes/Executive vote - April 18, 2024.md

```diff
@@ -0,0 +1,128 @@
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/march-2024-aligned-delegate-compensation/24088)
+
+If this executive proposal passes, the following transfers totalling **214.55 MKR** for Aligned Delegate Compensation will be made to qualifying Aligned Delegates.
+
+| Delegate | Address | Amount (MKR) |
+|--|--|--|
+| BONAPUBLICA | 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3 | 41.67 |
+| Cloaky | 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818 | 41.67 |
+| TRUE NAME | 0x612F7924c367575a0Edf21333D96b15F1B345A5d | 41.67 |
+| BLUE | 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf | 39.75 |
+| vigilant | 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61 | 13.89 |
+| Pipkin | 0x0E661eFE390aE39f90a58b04CF891044e56DEDB7 | 13.89 |
+| JAG | 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f | 9.08 |
+| UPMaker | 0xbB819DF169670DC71A16F58F55956FE642cc6BcD | 12.93 |
+
+### AVC Members Compensation Q1 2024
+
+- **Authorization:** [Atlas Section 2.5.10](https://mips.makerdao.com/mips/details/MIP101#2-5-10-avc-member-participation-rewards)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/avc-member-participation-rewards-q1-2024/24083)
+
+If this executive proposal passes, the following distributions totalling **175.14 MKR** will be made for Q1 2024 AVC member compensation.
+
+| AVC Member | Address | Amount (MKR) |
+|--|--|--|
+| IamMeeoh | 0x47f7A5d8D27f259582097E1eE59a07a816982AE9 | 20.85  |
+| DAI-Vinci | 0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a | 20.85  |
+| opensky | 0xf44f97f4113759E0a57756bE49C0655d490Cf19F | 20.85  |
+| ACRE DAOs | 0xBF9226345F601150F64Ea4fEaAE7E40530763cbd | 20.85  |
+| Res | 0x8c5c8d76372954922400e4654AF7694e158AB784 | 20.85  |
+| Harmony | 0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3 | 20.85  |
+| Libertas | 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f | 20.85  |
+| seedlatam.eth | 0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299 | 20.85  |
+| 0xRoot | 0xC74392777443a11Dc26Ce8A3D934370514F38A91 | 8.34  |
+
+### Aave Revenue Share Payment
+
+- **Authorization:** [Support Scope Section 9.4.1](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-3-q1-2024/24014)
+
+If this executive proposal passes, a payment from the Surplus Buffer of **238,339 DAI** to a smart contract ([0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c)) controlled by Aave Governance will be made as part of the Spark Protocol-Aave Revenue Share.
+
+### Whitelisting New Address in the RWA015-A Output Conduit
+
+- **Authorization:** [Atlas Section 2.5.10](https://mips.makerdao.com/mips/details/MIP101#2-5-10-avc-member-participation-rewards)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084)
```

Comment:
:red_circle: **blocking**

1. Detail of address to whitelist is missing
2. Authorization link is incorrect